examples/process_gain.py

```diff
@@ -127,14 +123,6 @@ def determine_power_from_fit(filename, guessamp, guessph):
     fl.plot(output_power, "o")
     fl.plot(output_spline(nu_fine))
     fl.next("Gain")
-    gain_dB = (
-        10 * np.log10(output_power / input_power)
-        + attenuator_dB
-    )
-    gain_dB.name("Gain").set_units("dB").set_plot_color(
-        "purple"
-    )
-    gain_dB.human_units(scale_data=True)
-    gain_spline = gain_dB.spline_lambda()
-    fl.plot(gain_spline(nu_fine))
-    fl.plot(gain_dB, "o")
+    gain_dB = 10 * np.log10(output_spline(nu_fine) / input_spline(nu_fine)) + attenuator_dB
+    gain_dB.name("Gain").set_units("dB").set_plot_color("purple")
+    fl.plot(gain_dB)
```

Comment:
avoids creating a new spline just divide the two premade splines and call it a day

Reply:
this also avoids the human_units thing I think

Reply:
yes -- but what motivated this is b/c I changed above to allow for different spacing, this also permits different spacing.